plugins/woocommerce/client/legacy/js/admin/wc-shipping-zones.js

```diff
@@ -169,7 +169,7 @@
 							}
 
 							$method_list.append(
-								'<li class="wc-shipping-zone-method ' + class_name + '">' + shipping_method.title + '</li>'
+								'<li class="wc-shipping-zone-method ' + class_name + '" data-id="' + shipping_method.instance_id + '">' + shipping_method.title + '</li>'
```

Comment:
I agree this can be more helpful for styling. But Alternatively, have you considered [nth_child](https://www.w3schools.com/cssref/sel_nth-child.php) CSS for styling it?

Reply:
nth_child will not allow me to target a specific shipping method.